Public/Src/Cache/ContentStore/Library/Sessions/ContentSessionBase.cs

```diff
@@ -80,6 +83,7 @@ protected ContentSessionBase(string name)
                     Tracer,
                     () => PinCoreAsync(operationContext, contentHash, urgencyHint, _counters[ContentSessionBaseCounters.PinRetries]),
                     traceOperationStarted: false,
+                    traceOperationFinished: TracePinFinished,

```

Comment:
Actually, I think we need to stop tracing the start.
Currently, we're adding 'extraEndMessage' so with your change we'll lose the hash altogether.

Reply:
And because we already skipping the start it means that this trace is completely skipped.

Which is fine, but definitely deserves a comment saying that we don't have to trace because of code duplication.

---
In reply to: [302249447](https://github.com/microsoft/BuildXL/pull/572#discussion_r302249447) [](ancestors = 302249447)

Reply:
I left a comment stating that in the override of TracePinFinished in BlobContentSession